returnn/tf/network.py

```diff
@@ -18,6 +18,12 @@
 from returnn.tf.util.basic import Data, DimensionTag, reuse_name_scope, VariableAssigner
 
 
+class DataNotFound(Exception):
+  """
+  Used for KeyErrors when accessing ExternData
```

Comment:
```suggestion
  When accessing non-existing ExternData data key
```
(Nothing about `KeyError` here. `KeyError` is totally irrelevant.)